frontend/awx/resources/projects/ProjectPage/ProjectDetails.tsx

```diff
@@ -10,28 +11,126 @@ import {
 } from '../../../../../framework';
 import { RouteObj } from '../../../../Routes';
 import { StatusCell } from '../../../../common/StatusCell';
+import { ExecutionEnvironmentDetail } from '../../../common/ExecutionEnvironmentDetail';
 import { ScmType } from '../../../../common/scm';
 import { Project } from '../../../interfaces/Project';
+import {
+  LabelGroup,
+  TextList,
+  TextListItem,
+  TextListItemVariants,
+  TextListVariants,
+} from '@patternfly/react-core';
+import { CredentialLabel } from '../../../common/CredentialLabel';
+import { StandardPopover } from '../../../../../framework/components/StandardPopover';
+import { useCallback } from 'react';
+import { useAwxWebSocketSubscription } from '../../../common/useAwxWebSocket';
+import { useGet } from '../../../../common/crud/useGet';
 
 export function ProjectDetails(props: { project: Project }) {
   const { t } = useTranslation();
   const { project } = props;
   const history = useNavigate();
+  {
+    /* TODO config provider */
+  }
+  const { data: config } = useSWR<IConfigData>(`/api/v2/config/`, (url: string) =>
+    fetch(url).then((r) => r.json())
+  );
+  const view = useGet<Project>(`/api/v2/projects/${project.id}/`);
+  const { refresh } = view;
+  const handleWebSocketMessage = useCallback(
+    (message?: { group_name?: string; type?: string }) => {
+      switch (message?.group_name) {
+        case 'jobs':
```

Comment:
since there's only one `case` here, it should probably be an `if` instead of a `switch`

Reply:
Agree, we should probably make this change throughout at a later point.